core/src/apps/common/cbor.py

```diff
@@ -43,7 +43,11 @@
 _CBOR_TRUE = const(0x15)
 _CBOR_NULL = const(0x16)
 _CBOR_BREAK = const(0x1F)
-_CBOR_RAW_TAG = const(0x18)
+
+
+# See https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml
+_CBOR_RAW_TAG = const(0x18)  # Tag 24
+_CBOR_SET_TAG = const(0x102)  # Tag 258
```

Comment:
What about tag 30? It is used in pool registration for unit_interval, so I'd perhaps expect it to appear in this list too?

Reply:
For whatever reason, all the cbor constants are private in this file. But yeah tag 30 is used in pool registration. Tag 24 is also used somewhere but, again, hardcoded instead of using the constant that already exists here